### PR TITLE
Frame counting

### PR DESCRIPTION
**Pull request by LabaBL**
Implemented number of frames trained on as the primary counter for the main loop of the algorithm.
Added number of frames to csv reporting.
Age is now disregarded when selection during a full evaluation is performed. 
Warmup period has been removed.

Known issue => Time estimation is off due to max_gen no longer defined.

### CODE REVIEW

File: src/selection/genetics.py [hunk 1]
```diff
@@ -45,7 +45,6 @@ def clone_agent(agent: Agent, agent_type: AgentType) -> Agent:
     clone.load_state_dict(agent.state_dict())
 
     # Copy training counters
-    clone.number_of_frames_evaluated = agent.number_of_frames_evaluated
     clone.number_of_mutations = agent.number_of_mutations
```
Comment (valsted) [inline src/selection/genetics.py:48]: Isn't this just it's age?
